layers/exporters/matrix_gen.py

```diff
@@ -100,14 +94,11 @@ def __init__(self, source='taxii', local=None):
                     self.collections[collection.title.split(' ')[0].lower()] = TAXIICollectionSource(tc)
         elif source.lower() == 'local':
             if local is not None:
-                try:
-                    self.collections['enterprise'] = FileSystemSource(local)
-                    self.collections['mobile'] = FileSystemSource(local)
-                except:
-                    raise BadLocation
```

Comment:
I removed the try-catch here because the FileSystemSource will raise its own exception if it has trouble, and their error will be more versatile than our BadLocation error (since theirs will presumably differentiate between invalid JSON, missing files, and invalid STIX.